src/Stripe.net/Entities/Charges/ChargePaymentMethodDetails/ChargePaymentMethodDetailsIdeal.cs

```diff
@@ -0,0 +1,20 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class ChargePaymentMethodDetailsIdeal : StripeEntity
+    {
+        [JsonProperty("bank_code")]
+        public string BankCode { get; set; }
+
+        [JsonProperty("bank_name")]
+        public string BankName { get; set; }
+
+        [JsonProperty("bic")]
+        public string Bic { get; set; }
+
+        [JsonProperty("verified_name")]
+        public string VerifiedName { get; set; }
```

Comment:
I think we got the schema from Giropay above instead. Ideal should have `iban_last4` and  `bank`